app/views/pages/api.html.erb

```diff
@@ -0,0 +1,19 @@
+<header>
+  <%= image_tag "Logo.png", id: "logo-big" %>
+  <h1 id="logo-title">Sortera Mera API</h1>
+</header>
+
+<main>
+  <article>
+    <h2>Nu även ett API!</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+      quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+      Besök <a href="#">dokumentationen</a> för att veta mer!
```

Comment:
I like to have `FIXME` or `TODO` comments when something is not done. In this case we may miss to add the documentation href later. But if you add a issue in the task manager for the project that will do as well.